### PR TITLE
POC for custom footer

### PR DESCRIPTION
**Pull request by peterVorman**

### CODE REVIEW

File: ckanext/opendata_theme/opengov_custom_footer/plugin/__init__.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import ckan.plugins as plugins
+import ckan.plugins.toolkit as toolkit
+from ckan.exceptions import CkanVersionException
+
+try:
+    toolkit.requires_ckan_version("2.9")
+except CkanVersionException:
+    from ckanext.opendata_theme.opengov_custom_footer.plugin.pylons_plugin import MixinPlugin
+    from webhelpers.html import literal
+    from ckanext.opendata_theme.opengov_custom_footer.controller import CustomFooterController
```
Comment (jguo144) [inline ckanext/opendata_theme/opengov_custom_footer/plugin/__init__.py:10]: There is an import error here as controller.py was renamed. Instead this should probably be `common_controller`.

File: ckanext/opendata_theme/opengov_custom_footer/templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<footer class="site-footer">
+  <div class="container">
+    {% block footer_content %}
+    {% set layout_type = h.get_footer_data('layout_type')|int %}
+    <div class="row">
+        {% for index in range(layout_type) %}
```
Comment (jguo144) [inline ckanext/opendata_theme/opengov_custom_footer/templates/footer.html:6]: For the default layout the default content should be displayed, but the template here is good enough for a POC.

File: ckanext/opendata_theme/opengov_custom_footer/plugin/__init__.py (hunk 1 above)
Comment (peterVorman) [reply ckanext/opendata_theme/opengov_custom_footer/plugin/__init__.py]: updated.

**Review by jguo144 — APPROVED** [review]
LGTM

**Pull request merged**